app/src/main/java/org/oppia/android/app/home/recentlyplayed/PromotedListAdapter.kt

```diff
@@ -14,8 +14,8 @@ import org.oppia.android.databinding.SectionTitleBinding
 private const val VIEW_TYPE_SECTION_TITLE_TEXT = 1
 private const val VIEW_TYPE_SECTION_STORY_ITEM = 2
 
-/** Adapter to inflate different items/views inside [RecyclerView] for Ongoing Story List. */
-class OngoingListAdapter(
+/** Adapter to inflate different items/views inside [RecyclerView] for Promoted Story List. */
```

Comment:
Recommend clarifying this comment:
`Adapter to inflate different Promoted Activity items/views inside the [RecentlyPlayedFragment]'s [RecyclerView].`

Reply:
done